### DISCUSSION TITLE
Welcome to net-ssh Discussions!

### DISCUSSION THREAD

**Discussion by mfazekas** [net-ssh/net-ssh#898] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by donoghuc** [comment]
Love this idea @mfazekas ! I work at Puppet, we use net-ssh in a project called [bolt](https://github.com/puppetlabs/bolt). I will be looking at moving on to the 7.x series of net-ssh soon!
